sidebars.js

```diff
@@ -141,17 +141,36 @@ const sidebars = {
     },
     {
       type: "category",
-      label: "Customize",
+      label: "Customize your Subnet",
       collapsed: true,
       items: [
         "subnets/customize-a-subnet",
+        "subnets/cross-subnet-communication"
+      ],
+    },
+    {
+      type: "category",
+      label: "Subnet-EVM Chain",
+      collapsed: true,
+      items: [
+        "subnets/customize-a-subnet",
```

Comment:
Is this how you intended for users?
<img width="862" alt="Screenshot 2023-06-01 at 3 38 22 PM" src="https://github.com/ava-labs/avalanche-docs/assets/12171681/66859370-10fd-4116-866e-3d620307ce42">

Reply:
Oops!  I wrote the wrong name on the side bar 🤦‍♀️. good catch!